### PR TITLE
make satellite more resilient to any user model

### PR DESCRIPTION
**Pull request by capripot**
- Create user with only attributes that exists on the model
- Rename `image_url` to `image`
- Fix nil exception when passing nil to `Addressable::URI`
- Remove outdated testing environment

### CODE REVIEW

**Review by MGerrior — COMMENTED** [review]
👍

File: spec/satellite/models/user_spec.rb [hunk 1]
```diff
@@ -42,7 +42,7 @@ def auth(options = {})
       user = User.find_or_create_with_omniauth(auth('info' => info))
       expect(user.name).to eq('Missy Wednesday')
       expect(user.email).to eq('missy@example.com')
-      expect(user.image_url).to eq('https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50')
+      expect(user.image).to eq('https://www.gravatar.com/avatar/205e460b479e2e5b48aec07710c08d50')
```
Comment (niuage) [inline spec/satellite/models/user_spec.rb:45]: In my opinion `image` is not the best choice as it's not what we use in platform nor copro.

Comment (niuage) [reply spec/satellite/models/user_spec.rb:45]: Ok, nevermind, I realized you used that because it's the name used in the omniauth schema. 👍

Comment (capripot) [reply spec/satellite/models/user_spec.rb:45]: yep

**Pull request merged**